### DISCUSSION TITLE
Feature Request: Only add movies or TVs above a certain rating

### DISCUSSION THREAD

**Discussion by j21w91** [meisnate12/Plex-Meta-Manager#440] [Q&A]
**Describe the solution you'd like**
This project is amazing. It’s taken my Plex to the next level. However, if I leave the lists unmonitored then it clogs up my storage with certain films I’d never watch. And none of my users would. 

I’d like an option to add an IF statement to the lists that would only ADS a film or tv series to radarr/sonarr if the TMDb or IMDb ratings are above a certain number. 

on top of that, if there’s some way to only add if the movie is not on Netflix/Amazon Prime UK would also be great. Most of my users already have Netflix. So again, those films just take up space and are annoying to have to manually remove one by one.

**Comment by meisnate12** [comment]
You can use [Collection Filters](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Collection-Filters) to filter what movies are available to be added to a collection and to what is missing from a collection.

I currently have no plans to add any netflix or amazon filter mainly because I don't have a source for that information. If you can provide a source that I can use that has all the movies/shows on a platform I would be glad to add it as a filter.
